go/vt/vtgate/evalengine/compiler_asm.go

```diff
@@ -2193,6 +2195,17 @@ func (asm *assembler) NullCheck1(j *jump) {
 	}, "NULLCHECK SP-1")
 }
 
+func (asm *assembler) NullCheck1r(j *jump) {
+	asm.emit(func(vm *VirtualMachine) int {
+		if vm.stack[vm.sp-1] == nil {
+			vm.stack[vm.sp-2] = nil
+			vm.sp--
+			return j.offset()
+		}
+		return 1
+	}, "NULLCHECK SP-1, SP-2")
```

Comment:
@vmg This looks indistinguishable for debugging from `NullCheck2` so a different description is probably better?